core/embed/trezorhal/stm32u5/mpu.c

```diff
@@ -307,6 +323,13 @@ mpu_mode_t mpu_reconfig(mpu_mode_t mode) {
     case MPU_MODE_SAES:
       SET_REGION( 5, PERIPH_BASE_NS,           PERIPH_SIZE,  PERIPHERAL,  YES,    YES ); // Peripherals - SAES, TAMP
       break;
+    case MPU_MODE_APP:
+      if (drv->unpriv_fb_addr != 0) {
+        SET_REGION( 5, drv->unpriv_fb_addr, drv->unpriv_fb_size,   SRAM,  YES,    YES ); // Frame buffer or display interface
```

Comment:
ad the comment, we don't actually expose anything else than framebuffes here, right?

Reply:
fixed, see 929ef8e8cf0561c7aefeb339395f49264f6ebe0b